pkgs/by-name/ph/photoprism/backend.nix

```
@@ -30,18 +40,17 @@ buildGoModule rec {
   ];
 
   postPatch = ''
-    substituteInPlace internal/commands/passwd.go --replace '/bin/stty' "${coreutils}/bin/stty"
+    substituteInPlace internal/commands/passwd.go --replace-error '/bin/stty' "${coreutils}/bin/stty"
```

```suggestion
    substituteInPlace internal/commands/passwd.go --replace-fail '/bin/stty' "${coreutils}/bin/stty"
```

Did the build succeed for you with this?

---

ah sorry I just applied it. Give me a second

---

nevermind. Sorry for the noise here.

I just thought it works, but sadly I am getting:

```
error: tensorflow-bin: unsupported configuration: x86_64-linux_313
```
related to #392602. will mark it as draft until it is fixed.